File: PCube/src/app/models/role.ts
export class Role {
    id: number;
    role_name : string; 
    access_level : number;      

    constructor(id, role_name, access_level){
        this.id = id;
        this.role_name = role_name;
        this.access_level = access_level;
    }
}

File: PCube-API/src/domain/role.py
class Role:

    def __init__(self):
        self.id = -1
        self.role_name = ""
        self.access_level = -1

    def asDictionary(self):
        return {
            "id": self.id,
            "role_name": self.role_name,
            "access_level": self.access_level
        }
